lib/merch_calendar.rb

```diff
@@ -1,12 +1,12 @@
+require "date"
 
 #
 module MerchCalendar
-
```

Comment:
not sure but feels cleaner to include the deprecation date in each respective calendar? Since then you won't need to do an extra require?